packages/spright-components/agents.md

```diff
@@ -0,0 +1,48 @@
+# Spright Components – AI Instructions
+
+## Overview
+
+Specialized components that serve specific domain needs while maintaining Nimble's quality standards.
+
+- **Prefix**: `spright-`
+- **Status**: Production-grade, domain-specific.
+
+## Build & Test
+
+Run these commands from the repo root:
+
+- **Build**: `npm run build -w @ni/spright-components`
+- **Test**: `npm run tdd:watch -w @ni/spright-components`
+
+## Key References
+
+- [`CONTRIBUTING.md`](CONTRIBUTING.md) – Ownership, code quality, and documentation requirements.
+
+## Requirements vs Nimble
+
+|                        | `nimble-components` | `spright-components` |
+| ---------------------- | :-----------------: | :------------------: |
+| Approved spec          |         🟢          |          🟢          |
+| Unit tests             |         🟢          |          🟢          |
+| Storybook visual tests |         🟢          |          🟢          |
+| Storybook API docs     |         🟢          |          🟢          |
+| Storybook usage docs   |         🟢          |          🟡          |
+| Approved VxD\*         |         🟢          |          🟡          |
+| Approved IxD\*         |         🟢          |          🟡          |
+| Angular/Blazor support |         🟢          |          🟡          |
+| Proper a11y            |         🟢          |          🟡          |
+| Minimal tech debt      |         🟢          |          🟡          |
+| Mobile support         |         🟡          |          🟡          |
+
+🟢 = required, 🟡 = optional\*By an interaction and/or visual designer
```

Comment:
outdated and not in this library, was moved to the root contributing. it should just not be duplicated